CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ x.x.x Release notes (yyyy-MM-dd)
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
 * ReactNative for Android no longer uses deprecated methods and can build using Gradle 5.0 and above. ([#1995](https://github.com/realm/realm-js/issues/1995))
+* Upgraded Realm Core to v5.12.1 and Realm Sync to v3.13.4 as a temporary fix due to a regression on Windows. ([#2169](https://github.com/realm/realm-js/issues/2169), since v2.21.0)
```

Comment:
```suggestion
* Downgraded Realm Core to v5.12.1 and Realm Sync to v3.13.4 as a temporary fix due to a regression on Windows. ([#2169](https://github.com/realm/realm-js/issues/2169), since v2.21.0)
```